rpcd/playbooks/roles/rpc_pre_upgrade/templates/status.txt.j2

```diff
@@ -25,6 +25,11 @@ cinder service-list output as at {{ ansible_date_time.date }}-{{ ansible_date_ti
 {{ line }}
 {% endfor %}
 
+orchestration service-list output as at {{ ansible_date_time.date }}-{{ ansible_date_time.time }}
```

Comment:
The command is incorrect:
s/orchestration service-list/openstack orchestration service list/